src/kirin/analysis/forward.py

```diff
@@ -35,6 +35,7 @@ def run_analysis(
         self,
         method: ir.Method,
         args: tuple[LatticeElemType, ...] | None = None,
+        raise_error_if_failed: bool = False,
```

Comment:
```suggestion
        no_raise: bool = True,
```

I think it's usually called this?